api/features/feature_external_resources/views.py

```diff
@@ -0,0 +1,37 @@
+from django.shortcuts import get_object_or_404
+from rest_framework import viewsets
+from rest_framework.exceptions import PermissionDenied
+from rest_framework.permissions import IsAuthenticated
+
+from features.models import Feature
+from features.permissions import FeatureStatePermissions
+
+from .models import FeatureExternalResource
+from .serializers import FeatureExternalResourceSerializer
+
+
+class FeatureExternalResourceViewSet(viewsets.ModelViewSet):
+    serializer_class = FeatureExternalResourceSerializer
+    permission_classes = [IsAuthenticated, FeatureStatePermissions]
+
+    def get_queryset(self):
+        if "pk" in self.kwargs:
+            return FeatureExternalResource.objects.filter(id=self.kwargs["pk"])
+        else:
+            features_pk = self.kwargs["feature_pk"]
+            return FeatureExternalResource.objects.filter(feature=features_pk)
+
+    def create(self, request, *args, **kwargs):
+        feature = get_object_or_404(
+            Feature.objects.filter(
+                id=self.kwargs["feature_pk"],
+            ),
+        )
+        if hasattr(feature.project, "github_config"):
+            raise PermissionDenied("You don't have a GitHub config")
```

Comment:
I think 400 would be better here instead of 403 (returned by permission denied)

Reply:
I'd just move this to the serializer and raise validation Error

Reply:
I prefer to keep it here and not move it to the serializer since it is not validating directly a field received in the request.